tools/tm-bench/transacter.go

```diff
@@ -75,19 +76,22 @@ func (t *transacter) Start() error {
 		t.conns[i] = c
 	}
 
-	t.wg.Add(2 * t.Connections)
+	t.startingWg.Add(t.Connections)
+	t.endingWg.Add(2 * t.Connections)
```

Comment:
now when you're working on this, shouldn't we only wait until we stop sending transactions (and not waiting for receive to finish)?

e.g. `t.endingWg.Add(2 * t.Connections)` => `t.endingWg.Add(t.Connections)`

Reply:
I thought we wait for receive to finish, so that we can close connections cleanly. (I.e. not try to receive on a closed channel)

If thats true, I think its fine if we wait for all connections to close before displaying output to the user. We need all connections to cleanly close. Computing output shouldn't take that long, so it may be weird if there is output but the program is still running for another second or two. (Assuming large numbers of connections)